action_msgs/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,109 @@
+This document is a declaration of software quality for the `action_msgs` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `action_msgs` Quality Declaration
+
+The package `action_msgs` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 4 in REP-2004](https://www.ros.org/reps/rep-2004.html).
```

Comment:
```suggestion
Below are the rationales, notes, and caveats for this claim, organized by each requirementlisted in the [Package Quality Categories in REP-2004](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-quality-categories) of the ROS2 developer guide.
```

Reply:
I'm currently linking to the non-existent rep page because the content in the developer guide is slated for removal.